resources/party.ts

```diff
@@ -181,12 +361,17 @@ export default class PartyTracker {
         nick: nick,
       };
     } else {
-      const jobName = Util.jobEnumToJob(partyMember.job);
-      const role = Util.jobToRole(jobName);
+      const job = Util.jobEnumToJob(partyMember.job);
+      const jobAbbr = jobLocalized[job]?.abbr?.[Options.AlertsLanguage ?? 'en'] ?? job;
+      const jobFull = jobLocalized[job]?.full?.[Options.AlertsLanguage ?? 'en'] ?? job;
+      const role = Util.jobToRole(job);
+      const roleName = roleLocalized[role]?.[Options.AlertsLanguage ?? 'en'] ?? role;
       ret = {
         id: partyMember.id,
-        job: jobName,
-        role: role,
+        job: job,
+        jobAbbr: jobAbbr,
+        jobFull: jobFull,
```

Comment:
Since the names and the variables are the same for object properties, you could use the simpler alternative representation for it:

```js
let ret = {
  job,
  jobAbbr,
  jobFull,
}
```

represent the same as following, but simpler:

```js
let ret = {
  job: job,
  jobAbbr: jobAbbr,
  jobFull: jobFull,
}
```

Reply:
I didn't use abbreviations because the original code didn't have any abbreviations, such as 'name' and 'nick,' so I followed this style.

Reply:
Well, in general we prefer the shorter style, since there isn't a lint rule forcing this style currently, it is also ok to not changing it just for now.

Reply:
Oh, I tried to abbreviate it, but it violates the project's ESLint rules "object-shorthand".

Reply:
Oh, it looks like we are forcing to not use the shorter style, I recall the wrong side, my apologies. Never mind.